src/controllers/UsersController.php

```diff
@@ -298,4 +298,38 @@ public function passwordReset(Request $request, $db)
         }
 
     }
+
+
+
+    public function deleteUser($request, $db)
+    {
+
+
+        if (! isset($request->user_id)) {
+            throw new Exception("You must be logged in to delete data", 400);
+        }
+        // delete the user
+        $user_id     = $this->getItemId($request);
+        $user_mapper = new UserMapper($db, $request);
+
+
+        $is_admin = $user_mapper->thisUserHasAdminOn($user_id);
+        if (! $is_admin) {
+            throw new Exception("You do not have permission to do that", 400);
+        }
+        
+        if ($user_mapper->delete($user_id)){
+            header("Content-Length: 0", null, 204);
+            exit; // no more content            
+        }else{
```

Comment:
You can omit the ```else``` here. 

When the ```if``` was used, the script ended with the ```exit``` :wink: